projects/mci/src/mci_output.c

```diff
@@ -91,6 +101,7 @@ StatusCode mci_output_init(MotorControllerOutputStorage *storage, Mcp2515Storage
   };
   storage->motor_can = motor_can;
   status_ok_or_return(pedal_rx_init(&storage->pedal_storage, &pedal_settings));
+  status_ok_or_return(regen_braking_init());
```

Comment:
IMO this should be initialized in main rather than here since the regen_braking module isn't really "owned" by the mci_output module - other modules could conceivably care about the regen braking state.

Reply:
Done. moved to main